install/collect_mac_addresses_for_ncns.md

```diff
@@ -3,24 +3,24 @@
 ## Topics
 
 1. [Set up networking](#1-set-up-networking)
-2. [Collect MAC addresses](#2-collect-mac-addresses)
+1. [Collect MAC addresses](#2-collect-mac-addresses)
 
 ## 1. Set up networking
 
 This assumes that the HMN is not setup on the PIT node, and these steps cater to bare-metal and configured switches.
 
 1. (`pit#`) Change into the preparation directory.
 
-   > **`NOTE`** If `PITDATA` is not defined, see [set reusable environment variables](./pre-installation.md#set-reusable-environment-variables).
+   > **`NOTE`** If `PITDATA` is not defined, see [set reusable environment variables](./pre-installation.md#15-set-reusable-environment-variables).
 
    ```bash
    cd ${PITDATA}/prep
    ```
 
 1. (`pit#`) Confirm that the `ncn_metadata.csv` file in this directory has the new information.
 
-   > **NOTE:** If the file is missing, please generate the file by following
-   > [Generate Topology Files](./pre-installation.md#generate-topology-files) after the
+   > **NOTE** If the file is missing, please generate the file by following
```

Comment:
Tech docs usually don't say "please", I don't think, as it just makes them longer.
```suggestion
   > **NOTE** If the file is missing, then generate the file by following
```